### PR TITLE
Getting started read me file for setup for new users.

### PR DESCRIPTION
**Pull request by mo-durvesh**
Getting started read me file, will need the actual readme to be updated to include a link to this doc.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+From the root of the project run:
+
+```shell script
+composer install
+```
+
+## Create a Database
+
+Next up, you need to create a database for the project. Both MySQL 5.5+ and PostgreSQL 9.5+ are supported.
```
Comment (davidrapson) [inline docs/getting-started.md:22]: Wonder if we should only specify MySQL as that's what we use in production?

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+If you’re not using MAMP or another localhosting tool, you will probably need to update your hosts file, so your computer knows to route requests to your chosen host name to the local computer.
+
+-   macOS/Linux/Unix: `/etc/hosts`
+-   Windows: `\Windows\System32\drivers\etc\hosts`
+
+You can test whether you set everything up correctly by pointing your web browser to `http://<Hostname>/index.php?p=admin/install` (substituting <Hostname> with your web server’s host name). If Craft’s Setup Wizard is shown, the host name is correctly resolving to your Craft installation.
+
+## Run the Setup Wizard
+Finally, it’s time to run Craft’s Setup Wizard. You can either run that from your [terminal](###Terminal Setup) or your [web browser](###Web Browser Setup).
```
Comment (davidrapson) [inline docs/getting-started.md:52]: Not sure if this link syntax will work on github `[terminal](###Terminal Setup)`

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+###Terminal Setup
```
Comment (davidrapson) [inline docs/getting-started.md:54]: This and the web browser set up heading need a space before the text to format correctly, worth double checking this by previewing the rendered file on github.

File: docs/getting-started.md (hunk 1 above)
Comment (mo-durvesh) [reply docs/getting-started.md:22]: I have changed it to only mention MySQL to be sure. I have also formatted the link syntax and added spacing to the headers. Thanks!

**Pull request merged**